src/Domain/FaceNormal.hpp

```diff
@@ -128,5 +169,55 @@ struct InterfaceCompute<Tags::BoundaryDirectionsExterior<VolumeDim>,
   using argument_tags = tmpl::list<Tags::Interface<dirs, Mesh<VolumeDim - 1>>,
                                    Tags::ElementMap<VolumeDim, Frame>>;
 };
+
+template <size_t VolumeDim>
+struct InterfaceCompute<Tags::BoundaryDirectionsExterior<VolumeDim>,
+                        UnnormalizedFaceNormalMovingMeshCompute<VolumeDim>>
+    : db::PrefixTag,
+      db::ComputeTag,
+      Tags::Interface<
+          Tags::BoundaryDirectionsExterior<VolumeDim>,
+          Tags::UnnormalizedFaceNormal<VolumeDim, Frame::Inertial>> {
+  using dirs = BoundaryDirectionsExterior<VolumeDim>;
+
+  static std::string name() noexcept {
+    return "BoundaryDirectionsExterior<UnnormalizedFaceNormal>";
+  }
+
+  static auto function(
+      const db::const_item_type<Tags::Interface<dirs, Mesh<VolumeDim - 1>>>&
+          meshes,
+      const ::ElementMap<VolumeDim, Frame::Grid>& logical_to_grid_map,
+      const domain::CoordinateMapBase<Frame::Grid, Frame::Inertial, VolumeDim>&
+          grid_to_inertial_map,
+      const double time,
+      const std::unordered_map<
+          std::string,
+          std::unique_ptr<domain::FunctionsOfTime::FunctionOfTime>>&
+          functions_of_time) noexcept {
+    std::unordered_map<::Direction<VolumeDim>,
+                       tnsr::i<DataVector, VolumeDim, Frame::Inertial>>
+        normals{};
+    for (const auto& direction_and_mesh : meshes) {
+      const auto& direction = direction_and_mesh.first;
+      const auto& mesh = direction_and_mesh.second;
+      auto internal_face_normal = unnormalized_face_normal(
+          mesh, logical_to_grid_map, grid_to_inertial_map, time,
+          functions_of_time, direction);
+      std::transform(internal_face_normal.begin(), internal_face_normal.end(),
+                     internal_face_normal.begin(), std::negate<>());
+      normals[direction] = std::move(internal_face_normal);
+    }
+    return normals;
+  }
```

Comment:
For my understanding: usually we try to have compute-items be thin wrappers around free functions... why not make this here (and the original without time-dep) a free function?

Reply:
Not having authored the original code I can't say for sure, but here's my understanding. Reversing the directions of the normals is super specific to the directions being considered and so isn't used generally, and thus factoring wouldn't reduce code. This might have changed with the moving mesh overload a bit, but I'm unsure how long this code will stick around for anyway.